jcl/source/windows/JclRegistry.pas

```diff
@@ -395,13 +395,14 @@ implementation
 uses
   {$IFDEF HAS_UNITSCOPE}
   System.SysUtils,
+  Winapi.AccCtrl,
   {$ELSE ~HAS_UNITSCOPE}
   SysUtils,
+  AccCtrl,
```

Comment:
Does this still compile under FPC?
It was in the ~FPC define below.

Reply:
It was in "not in ~FPC" define

Reply:
Actually you are right, it may affect FPC, although I'm not sure if it worked before, since JwaAccCtrl is commented out.